docs/applications/emby.md

```diff
@@ -0,0 +1,107 @@
+# Emby
+
+Homepage: [https://emby.media/](https://emby.media/)
+
+Emby is a mostly open-source media server with a client-server model. This
+install for Ansible-NAS provides a server, which various clients can then
+connect to from various platforms - other computers, smartphones, smart TVs,
+etc.
+
+Note that [Plex](https://www.plex.tv/), also included in Ansible-NAS, has a very
+similar functionality.
+
+## Usage
+
+Set `emby_enabled: true` in your `group_vars/all.yml` file. There are further
+parameters you can edit such as `movies_root` and `tv_root` lower down in this
+file. 
+
+## Specific Configuration
+
+The emby web interface can be found at port 8096 (http) or 8920 (https, if
+configured) of your NAS. After installation, point your browser there to start
+the configuration wizard. For instance, on a NAS with the IPv4 address
+192.168.1.2, browse to `http://192.168.1.2:8096`. Heimdall has a dedicated icon
+for emby. 
+
+By default, Ansible-NAS gives emby read/write access to the folders where your
```

Comment:
One thing I'm not willing to encourage people to do is edit the task files - primarily because it makes pulling latest changes difficult WRT to merge conflicts. I don't think there's any value in setting a volume to RO, but would accept a PR to move the config out to the all.yml if you think there's value in it.

Reply:
(Agree on the text changes above) The idea is to make absolutely sure that neither Plex nor Emby nor anybody else gets the idea that they can delete my media files (and that's just me being paranoid, go over to /r/datahorder on Reddit and check out their stuff). Of course they're RO on a file level, but still. -- Give me few days for the PR, will move it to all.yml.